x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/rule_editor_panel/json_rule_editor.test.tsx

```diff
@@ -12,25 +12,27 @@ import 'brace/mode/json';
 // warnings in the console which adds unnecessary noise to the test output.
 import '@kbn/test/target_node/jest/utils/stub_web_worker';
 
-import { EuiCodeEditor } from '@elastic/eui';
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 
-import { mountWithIntl } from '@kbn/test/jest';
-import { coreMock } from 'src/core/public/mocks';
-import { KibanaContextProvider } from 'src/plugins/kibana_react/public';
+import type { monaco } from '@kbn/monaco';
+import { shallowWithIntl } from '@kbn/test/jest';
+import { CodeEditor } from 'src/plugins/kibana_react/public';
```

Comment:
Have you tried using `CodeEditorField` component? It renders the code editor like a field with the correct border and padding so it looks a lot nicer than the plain code editor.

Reply:
Nice idea, I'll change it!

Side by side comparison before/after
![image](https://user-images.githubusercontent.com/5295965/128742834-594d728e-ca87-4d80-88b4-2cba18542c5e.png)

Reply:
Nice, looking much better! :)